gen/build_deploy/powershell.py

```diff
@@ -0,0 +1,49 @@
+"""
+Generates a powershell script to install Windows agent - dcos_install.ps1
+"""
+
+import os
+import os.path
+
+import gen.build_deploy.util as util
+import gen.template
+import gen.util
+
+import pkgpanda
+import pkgpanda.util
+
+
+def generate(gen_out, output_dir):
+    print("Generating Powershell configuration files for DC/OS")
+    make_powershell(gen_out, output_dir)
+
+
+def make_powershell(gen_out, output_dir):
+    """Build powershell deployment script and store this at Bootstrap serve"""
+
+    output_dir = output_dir + '/windows/prerequisites/'
+    pkgpanda.util.make_directory(output_dir)
+
+    bootstrap_url = gen_out.arguments['bootstrap_url']
+    if gen_out.arguments['master_discovery'] == 'static':
+        master_list = gen_out.arguments['master_list']
+    elif gen_out.arguments['master_discovery'] == 'master_http_loadbalancer':
+        master_list = gen_out.arguments['exhibitor_address'] + ':2181'
+    else:
+        master_list = 'zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181'
+
+    powershell_template_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                            'powershell/dcos_install.ps1.in')
+    with open(powershell_template_path, 'r') as f:
```

Comment:
:+1: I much prefer reading in a template to embedding the template in the Python code. Not in this PR, but we should change the Linux gen to do this too.